Examples/Tests/langmuir/PICMI_inputs_rz.py

```diff
@@ -51,7 +51,7 @@
 ##########################
 
 uniform_plasma = picmi.UniformDistribution(density = density,
-                                           upper_bound = [+18e-6, +18e-6, None],
+                                           upper_bound = [+18e-6, None, +40e-6],
```

Comment:
This was a bug found in this input file that made this test fail with the updated code.